packages/opds-web-client/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v0.3.2
+
+- Pass the redux store down the tree via context (this was not done previously)
+- Extracted PathForContext from the Root, which provides context via both legacy and new context APIs
+- Created a RouterContext component to be used by `circulation-patron-web` to provide router context.
```

Comment:
I think "this was not done previously" can be removed or rephrase that _now_ the redux store is passed down the tree. Also, it would be good to generalize and say that any application can use `RouterContext` and an example is `circulation-patron-web`, since this component should be able to be used by more applications.